win/pkgs/cuda.ps1

```diff
@@ -25,3 +25,26 @@ Write-Host "Installing CUDA..."
 & "${DownloadDir}/${exe}" -s | Out-Null
 
 ${Env:CUDA_PATH}=[System.Environment]::GetEnvironmentVariable("CUDA_PATH","Machine")
+
+# CUDA MSBuild integration is not working for VS BuildTools.
+# Manually install CUDA Build Customizations files into MSBuild customization folder.
+#
+# See Table 4. CUDA Visual Studio .props locations from
```

Comment:
Add `:` at the end, to make it a more natural line-breaking.

Reply:
fixed.